src/utils/HTTPUtils.ts

```diff
@@ -13,6 +14,37 @@ class HTTPUtils {
     private constructor() {
     }
 
+    static isTestContainersRequest(options: any, host: string): boolean {
+
+        if (!options || !host) {
+            return false;
+        }
+
+        const {
+            path,
+            socketPath,
+        } = options;
+
+        if (!path || !socketPath) {
+            return false;
+        }
+
+        if (host !== 'localhost'
+            || socketPath !== '/var/run/docker.sock') {
+            return false;
+        }
+
+        if (TESTCONTAINERS_HTTP_PATH_PATTERNS.PATTERN1.test(path)
```

Comment:
I am not sure whether it is possible that we might ignore non-test container requests because of first fragment of the path. Doesn't ignoring `/var/run/docker.sock` is necessary?